server/config.ts

```diff
@@ -82,6 +90,10 @@ export default {
       agent: new AgentConfig(Number(get('EXAMPLE_API_TIMEOUT_RESPONSE', 5000))),
     },
   },
+  serviceUrls: {
+    digitalPrison: get('DPS_HOME_PAGE_URL', 'http://localhost:3001', requiredInProduction),
+    prisonerProfile: get('PRISONER_PROFILE_URL', 'http://localhost:3001', requiredInProduction),
```

Comment:
Should these defaults be using different port numbers?

Reply:
I don't think it matters, as they are only used in the frontend-components to render the link when the crown is clicked or a breadcrumb to DPS. For local running or k8s, these are set in the .env and values files.